### DISCUSSION TITLE
Create a plain title slide instead of a regular one?

### DISCUSSION THREAD

**Discussion by henriqpsantos** [jgm/pandoc#9543] [Q&A]
I'm using a template that has footers on every beamer slide.

How can I make it so that the title slide has no footers? I've tried to generate a title slide with `\titleslide` within `# {.plain}`, which generates a title slide after the main title and toc slides.

I've searched the internet far and wide, to no avail. Only thing I've come across is `title-slide-attributes`, and adding plain to these doesn't work.

I'm using markdown input and pdf output through tectonic, if that matters.

**Comment by mikerenfro** [comment]
This isn't the most elegant fix, but to get a plain title frame, you can:

1. Run `pandoc -D beamer > template.beamer` to get an editable copy of the default Beamer template.
2. Then you can modify the line: `\frame{\titlepage}` to: `\frame[plain]{\titlepage}`.
3. Finally, you can either add `template: template.beamer` to your defaults file, or add `--template template.beamer` to your command-line parameters.
